uber/config.py

```diff
@@ -412,3 +412,9 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+if c.KIOSK_CC_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.STRIPE, "Pay with credit card now (faster)"])
+
+if c.GROUPS_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.GROUP, "Taking an unassigned Group badge (group leader must be present)"])
```

Comment:
Fair enough!  Although it occurs to me that since setting an option's value to the empty string already removes it from its ``_OPTS`` list and dictionary, in theory we could replace all of this with just setting that config in ``development.ini``.  The main downside to that is that it's less explicit than having an actual setting called ``kiosk_cc_enabled``, so this approach might be better for that reason.